truss/templates/server/model_wrapper.py

```diff
@@ -368,3 +360,30 @@ def _signature_accepts_kwargs(signature: inspect.Signature) -> bool:
 
 def _elapsed_ms(since_micro_seconds: float) -> int:
     return int((time.perf_counter() - since_micro_seconds) * 1000)
+
+
+def _handle_exception():
+    logging.exception("Exception while running predict")
+    raise HTTPException(
+        status_code=500, detail={"message": "Error while running predict"}
```

Comment:
nit: technically this can be an exception in postprocess. Ideally this just be "Internal Server Error" and the rest can be debugged from logs